include/net/fota_download.h

```diff
@@ -76,12 +76,14 @@ int fota_download_init(fota_download_callback_t client_callback);
  * @param host Hostname which you should start downloading from.
  * @param file Filepath to the file you wish to download.
  * @param sec_tag Security tag you want to use with HTTPS set to -1 to Disable.
+ * @param port Port which to connect to. Set to 0 for default HTTP/HTTPS port.
  *
  * @retval 0	     If download has started successfully.
  * @retval -EALREADY If download is already ongoing.
  *                   Otherwise, a negative value is returned.
  */
-int fota_download_start(const char *host, const char *file, int sec_tag);
+int fota_download_start(const char *host, const char *file, int sec_tag,
+			int port);
```

Comment:
Could you let `port` be an `u16_t` instead? that way it's the same type as the field in `download_client_cfg`.

Reply:
Ah, yes, of course! Thanks.